docs/cardinal/game/system/overview.mdx

```diff
@@ -121,39 +121,36 @@ package system
 // This provides an example of a system that doesn't rely on a message to update a component.
 func RegenSystem(world cardinal.WorldContext) error {
 	// Searches for all entities that have the Player and Health component.
-	search, err := world.NewSearch(cardinal.Exact(component.Player{}, component.Health{}))
```

Comment:
This is not related to your changes, but in this file there is a line `system/system_regen.go` somewhere, I think we should just name it `system/regen.go`, I would say the same for any other instances of file names in this file that are like that (I think there are a couple others)

Reply:
These prefixes match the file names in the starter-game-template. I think the sample files here and the files in the starget-game-template should match. I'm on board with removing the directory prefixes, but I think that's out of scope for this PR. Feel free to add a triage ticket.

Reply:
Agreed, made a ticket `WORLD-879`